pkg/patterns/declarative/pkg/applier/applylib.go

```diff
@@ -97,3 +123,13 @@ func (a *ApplySetApplier) Apply(ctx context.Context, opt ApplierOptions) error {
 
 	return nil
 }
+
+// NewParentRef maps a declarative object's information to the ParentRef defined in the applyset library.
+func NewParentRef(restMapper meta.RESTMapper, object runtime.Object, name, namespace string) applyset.Parent {
+	gvk := object.GetObjectKind().GroupVersionKind()
+	restMapping, err := restMapper.RESTMapping(gvk.GroupKind(), gvk.Version)
+	if err != nil {
+		return nil
```

Comment:
Return an error, I think

Reply:
done. This method is used to initialize the `ApplierOptions` which does not require Parent (unless the applier is `ApplysetApplier`) . But I think you are right, even if the parent is not used, its RESTMapping should still exist.